boards/kernel_layout.ld

```diff
@@ -65,6 +65,16 @@ SECTIONS
          _estack = .;
     } > ram
 
+    .interrupt :
+    {
+      KEEP(*(.vectors .vectors.*))
+      KEEP(*(.irqs))
+    } > ivt
+
+    .flash_config :
+    {
+      KEEP(*(.FlashConfig .FlashConfig.*))
+    } > fcf
```

Comment:
Rather than editing `kernel_layout.ld`, you might copy it to `s32k144_evb` and change the path specified in https://github.com/Ion-Mobility/tock/blob/8ac924e07f5d7daf15ffebf9bd96135210cca9c4/boards/s32k144_evb/layout.ld#L16 -- that way you won't have issues if you try to build it for other boards, and it will make it easier to pull in newer versions.